packages/common/store/index.js

```diff
@@ -1,17 +1,20 @@
 import {
   createStore, combineReducers, applyMiddleware, compose,
 } from 'redux';
-
+import storage from 'redux-persist/lib/storage';
+import { persistStore } from 'redux-persist';
```

Comment:
```suggestion
import { persistStore, storage } from 'redux-persist';
```

Reply:
also could you explain bit about usage of `redux-persist` and what we are trying to persist? so that other devs also know about it 🙏🏼

Reply:
storage isn't export in the root component 
we can use [Window.localStorage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage) but in any application is forbidden to interact global object like the window we access in from the libraries 
redux persist is the libraries to persist redux state to any storage such as localStorage, react-native async storage [more info](https://github.com/rt2zz/redux-persist)